docs/FAQ.md

```diff
@@ -0,0 +1,32 @@
+This page is for frequent asked questions and answers.
+
+
+### tmp folder fulled
+NNICTL will use tmp folder as a temporary folder to copy files under codeDir when executing experimentation creation. 
+When met errors like below, try clean up **tmp** folder first.
```

Comment:
try to clean up...?

Reply:
thanks, will modify.
Sorry for not checking this file since it is directly copied from WIKI